perf_benchmark/benchmark_common_test.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+#include "gtest/gtest.h"
+#include "perf_benchmark/benchmark_common.h"
+#include "absl/strings/escaping.h"
+
+namespace google {
+namespace grpc {
+namespace transcoding {
+
+namespace perf_benchmark {
+
+TEST(BenchmarkCommonTest, GetRandomBytesString) {
```

Comment:
Let's add some unit tests for the input stream. I'm not 100% convinced by the logic.